packages/dashboard/src/migration/convert-monitor-to-app-defintion.ts

```diff
@@ -0,0 +1,509 @@
+import {
+  DashboardConfiguration,
+  DashboardWidget,
+  MigrateDashboard,
+} from '../types';
+import {
+  DashboardWidgetType,
+  MonitorWidgetType,
+  MonitorMetric,
+  SiteWiseMonitorDashboardDefinition,
+  MonitorAnnotations,
+  MonitorWidget,
+} from './types';
+import { nanoid } from 'nanoid';
+import { colorPalette } from '@iot-app-kit/core-util';
+import { randomUUID } from 'crypto';
+
+interface ApplicationProperty {
+  propertyId: string;
+  aggregationType?: string;
+  resolution?: string;
+  refId?: string;
+  color?: string;
+}
+
+interface ApplicationAsset {
+  assetId: string;
+  properties: ApplicationProperty[];
+}
+
+interface ApplicationQuery {
+  properties: string[];
+  assets: ApplicationAsset[];
+}
+
+interface ApplicationQueryConfig {
+  source: string;
+  query: ApplicationQuery;
+}
+
+interface QueryConfig {
+  queryConfig: ApplicationQueryConfig;
+}
```

Comment:
Why are some types in this file and some in the types.ts file? How do you choose where they go?

Reply:
fixed, moved everything to one type file for anything being used in folder